cost/azure/hybrid_use_benefit/README.md

```diff
@@ -18,17 +18,17 @@ This Policy Template is used to automatically apply the Azure Hybrid Use Benefit
 
```

Comment:
Should add `Microsoft.Compute/virtualMachines/write` here as well.

Reply:
Added Microsoft.Compute/virtualMachines/write.